monailabel/utils/datastore/dicom/cache.py

```diff
@@ -54,12 +54,10 @@ def get_image_uri(self, image_id: str) -> str:
         if not os.path.exists(image_dir) or not os.listdir(image_dir):
             dicom_web_download_series(None, image_id, image_dir, self._client)
 
-        # TODO:: Do we need to convert always? For clients like OHIF it's not needed.
         image_nii_gz = os.path.realpath(os.path.join(self._datastore.image_path(), f"{image_id}.nii.gz"))
```

Comment:
we could add a client identifieror "do nii convesion flag" (based on the MONAILabel extension for the client) in the request if you would like to avoid this section from being executed if not required